codespell_lib/data/dictionary.txt

```diff
@@ -24916,16 +25010,28 @@ formattind->formatting
 formattings->formatting
 formattring->formatting
 formattted->formatted
+formattter->formatter
+formattters->formatters
 formattting->formatting
 formelly->formerly
 formely->formerly
 formen->foremen
 formend->formed
 formes->forms, formed,
 formidible->formidable
+formmat->format
+formmats->formats
 formmatted->formatted
+formmatter->formatter
+formmatters->formatters
+formmatting->formatting
 formost->foremost
 formt->format
```

Comment:
Is `formt` never a typo of `form`?

Reply:
`formt->form` exists, but is rare. I will add it.